### DISCUSSION TITLE
Mac packaging - can files, etc be left in Resources folder within app bundle?

### DISCUSSION THREAD

**Discussion by Blatman** [wailsapp/wails#1757] [Q&A] [answered]
Just been having a 'go' with WAILS and it seems that the application files from the frontend folder are always packaged within the MacOS binary leaving just the appicon.icns file in the Resources folder. I tried the -noPackage option in the config but it just builds the MacOS unix executable binary as normal but doesn't bundle it in a .app package. Is there some way of having the MacOS binary just containing the requisite go/wails runtimes and just pointing to the Resources folder with the application source files.

I also tried the -s switch (skip frontend build) but the app bundle just built as normal. Does the -frontend "path" flag maybe have anything to do with this scenario or will the app files always be placed within the app binary?

Be good to see a multi-window capability for desktop use and I need to conquer GO to see how the native menus work too.

**Comment by leaanthony** [comment, marked as answer]
Hi 👋 We currently don't support resources in the usual way with Mac. What do you see as the use case for this?

**Reply by Blatman** [reply]
Thanks for the answer Lea - no particular use case but I find it handy to be able to see the content of the app - not a showstopper and understandable that commercial apps will want the code obscured. I'm used to JS with NWjs and Neutralino so GO will be a challenge for me to work out

**Reply by leaanthony** [reply]
You'll be fine with Go. It's much easier to understand than `this` in JavaScript
